eve/main.yml

```diff
@@ -2,18 +2,31 @@
 version: 0.2
 
 branches:
-  feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*, ft/*, bf/*, chore/*, docs/*:
+  feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*:
     stage: "pre-merge"
+  development/*:
+    stage: "post-merge"
+
 
 stages:
   pre-merge:
-    worker:
+    worker: &pod
+      type: kube_pod
+      path: eve/workers/pod.yml
+    steps:
+    - TriggerStages:
+        name: trigger all the tests
+        stage_names:
+        - run-tests
+        - docker-build
```

Comment:
I thought we we only doing the docker-build post-merge and not during pre-merge ??

Reply:
Nevermind! Just saw the not about testing to see if it will build properly =)

Reply:
@ssalaues Thank you